integration/modeshape-jbossas-integration-tests/src/test/java/org/modeshape/test/integration/QueryIntegrationTest.java

```diff
@@ -54,14 +54,15 @@
     @Deployment
     public static WebArchive createDeployment() {
         File[] testDeps = Maven.configureResolver()
-                               .workOffline()
-                               .loadPomFromFile("pom.xml")
-                               .resolve("org.modeshape:modeshape-jcr:test-jar:tests:?").withTransitivity().asFile();
-                               
+                .workOffline()
+                .loadPomFromFile("pom.xml")
+                .resolve("org.modeshape:modeshape-jcr:test-jar:tests:?").withTransitivity().asFile();
+
         return ShrinkWrap.create(WebArchive.class, "query-test.war").addAsLibraries(testDeps)
-                         .addAsWebInfResource(EmptyAsset.INSTANCE, ArchivePaths.create("beans.xml"))
-                         .addAsResource(new File("src/test/resources/text-extractor/text-file.txt"))
-                         .setManifest(new File("src/main/webapp/META-INF/MANIFEST.MF"));
+                .addAsWebInfResource(EmptyAsset.INSTANCE, ArchivePaths.create("beans.xml"))
+                .addAsWebInfResource(new File("src/main/webapp/WEB-INF/jboss-deployment-structure.xml"), ArchivePaths.create("jboss-deployment-structure.xml"))
```

Comment:
since the ES test have been moved to another class, this isn't required here, right ?